templates/web/common/includes/display_product_api.tt.html

```diff
@@ -0,0 +1,32 @@
+
+[% IF single_param("jqm") %]
```

Comment:
`single_param` function is not available in templates @deveshidwivedi, so better compute the value in Display.pm and add the value it as a parameter (eg. has_jqm_param).